themes/default/content/docs/intro/console/auditing.md

```diff
@@ -9,11 +9,9 @@ aliases:
 - /docs/intro/console/collaboration/auditing/
 ---
 
-<div class="note note-info" role="alert">
-    <p>
-        Audit Logs are only available in <strong>Pulumi Enterprise</strong>.
-    </p>
-</div>
+{{% notes "info" %}}
+Audit Logs are only available in <strong>Pulumi Enterprise</strong>.
```

Comment:
```suggestion
Audit Logs are only available in **Pulumi Enterprise**.
```

Reply:
i fixed this to use a link to the plans instead.